tests/foreman/ui/test_myaccount.py

```diff
@@ -67,38 +67,13 @@ def tearDown(self):
         super(MyAccountTestCase, self).tearDown()
         self.account_user.delete()
 
-    def logged_test_user(self):
+    def session_test_user(self):
```

Comment:
Still, not sure about that piece of code as everywhere we just use direct `with Session`